job_templates/sag_lightning/config_fed_client.conf

```diff
@@ -0,0 +1,36 @@
+{
+  "format_version": 2,
+  # This is application scripts which will be invoked.
+  # Client can replaced this script with user's own traning script.
```

Comment:
(in multiple places) change to:

This is the application script which will be invoked.
Client can replace this script with user's own training script.